public/app/features/dashboard/containers/DashboardPageProxy.tsx

```diff
@@ -18,6 +18,10 @@ export type DashboardPageProxyProps = GrafanaRouteComponentProps<
 // This proxy component is used for Dashboard -> Scenes migration.
 // It will render DashboardScenePage if the user is only allowed to view the dashboard.
 function DashboardPageProxy(props: DashboardPageProxyProps) {
+  if (config.featureToggles.dashboardScene) {
```

Comment:
I imagine this feature toggle right now is only for testing purposes, but in the future, after implementing edit mode and dashboard settings, do you see it as the replacement for `dashboardSceneForViewers`? if yes, should we add some simple unit tests as we did with the other one?